docs/config.rst

```diff
@@ -174,9 +286,51 @@ Java Kafka clients.
   * Default: 1000
   * Importance: low
 
+``simpleconsumer.max.poll.records``
```

Comment:
Would there ever be any reason for this to be different from `simpleconsumer.cache.max.records`?